compiler/util/llvmDebug.cpp

```diff
@@ -124,14 +125,14 @@ LLVM_DITYPE debug_data::construct_type(Type *type)
   const char* name = type->symbol->name;
   ModuleSymbol* defModule = type->symbol->getModule();
   const char* defFile = type->symbol->fname();
-  if (strstr(defFile, "/modules/")!=NULL || strcmp(defFile, "<internal>")==0) {
+  /*if (strstr(defFile, "/modules/")!=NULL || strcmp(defFile, "<internal>")==0) {
 #if HAVE_LLVM_VER >= 37
     return NULL;
 #else
     return llvm::DIType();
 #endif
 
-  }
+  }*/
```

Comment:
I suspect (but havn't checked) that these strcmps were added to get this DebugInfo generation working with LLVM 3.7.